classes/01class/exercises/c01-aws01/fabioacp/ANSWER.md

```diff
@@ -0,0 +1,85 @@
+# C01-AWS01
+
+## Commands Execution Output
+
+- Commands to create the first EC2 instance and any additional resource required:
+```bash
+# Create a KeyPair
+aws ec2 create-key-pair --key-name MyKeyPair --query 'KeyMaterial' --output text > MyKeyPair.pem
+
+# Create a SG
+❯ aws ec2 create-security-group --group-name my-sg-public --description "SG for ec2 with public IP address" --vpc-id vpc-8beeebec
+{
+    "GroupId": "sg-0343908a8d3b8475e"
+}
+
+# You can start by confirming that your public address shows as included in the CIDR range 203.0.113.0/24.
+❯ curl https://checkip.amazonaws.com
+203.0.113.57
+
+# The following command adds another rule to enable SSH to instances in the same security group.
+aws ec2 authorize-security-group-ingress --group-id sg-903004f8 --protocol tcp --port 22 --cidr 203.0.113.0/24
```

Comment:
group-id doesnt match the one created in line 11.

Reply:
fixed